docker/spark_health.sh

```diff
@@ -0,0 +1 @@
+curl -s -k -o /dev/null -w "%{http_code}" -L https://$(hostname):$PORT2
```

Comment:
This works fine without `#!/bin/bash` shebang, but might be a good idea to add it anyway.

Reply:
+1